content/blog/18-19/g2_blogpost.md

```diff
@@ -0,0 +1,94 @@
+---
+title: "A Gateway to collaboration"
+created_at: 26-02-2018
+description: Het ProjectenProject
+author: Arne Bertrand
+---
+"Ik wil meedoen met een zeus project, hoe doe?"
+
+Het is een vraag die vervelend vaak voorkomt en verbazend moeilijk te 
+beantwoorden is. De keuze van het project op zich is vaak al lastig. 
+Welke technologie wil ik gebruiken, welke projecten zijn nog relevant,
+welk project heeft het meeste hulp nodig, zijn stuk voor stuk aspecten
+die meespelen en elkaar vaak tegenspreken.
+
+Eens een project gekozen, komt er vaak nog een hele boterham setup bij kijken
```

Comment:
Eens **je** een project gekozen **hebt**

Reply:
Hmm tis een stijlkeuze, en ik ben nogal fan van die constructie. Hoewel je eigenlijk wel gelijk hebt, persoonlijk is vrijwel altijd beter